packages/core/src/components/tag/_overrides.scss

```diff
@@ -157,7 +157,7 @@
     }
 
     // Interactive
-    &.#{$ns}-interactive {
+    &.#{$ns}-interactive:not(.#{$ns}-g-disabled) {
```

Comment:
It was introduced here: https://github.com/graphext/blueprint/commit/fbd8b6dc65ae8fefcac30467282a51f84fdf94d2

Was it a bug or something desired? @rauldiazpoblete

Reply:
I really don't remember that. :sweat_smile: 
From the description of the commit I understand that it was intentional, but if design wants to remove the interaction for disabled, go ahead.